### PR TITLE
MMCA-4278 Update SUB09 response as per the latest spec changes

### PR DESCRIPTION
**Pull request by harshare**

### CODE REVIEW

**Review by AdamCoops — APPROVED** [review]

File: app/domain/sub09/SubscriptionDisplayResponse.scala [hunk 1]
```diff
@@ -46,6 +46,11 @@ case class ContactInformation (
                                 emailVerificationTimestamp: Option[String]
                               )
 
+case class vatIDs (
```
Comment (tyagikumarvineet) [inline app/domain/sub09/SubscriptionDisplayResponse.scala:49]: Could you please rename the class name to VatID or VatId as case class name should be in UpperCamelCase?

File: app/domain/sub09/SubscriptionDisplayResponse.scala [hunk 2]
```diff
@@ -96,6 +104,7 @@ object SubscriptionResponse {
   implicit val euVatFormat = Json.format[EUVATNumber]
   implicit val xiSubscriptionFormat = Json.format[XiSubscription]
   implicit val returnParametersFormat = Json.format[ReturnParameters]
+  implicit val VATIDsFormat = Json.format[vatIDs]
```
Comment (tyagikumarvineet) [inline app/domain/sub09/SubscriptionDisplayResponse.scala:107]: better to rename it to vatIDFormat as val should follow the lowerCamelCase

File: app/domain/sub09/SubscriptionDisplayResponse.scala (hunk 1 above)
Comment (harshare) [reply app/domain/sub09/SubscriptionDisplayResponse.scala:49]: Renamed it to VatId

File: app/domain/sub09/SubscriptionDisplayResponse.scala (hunk 2 above)
Comment (harshare) [reply app/domain/sub09/SubscriptionDisplayResponse.scala:107]: Renamed it to vatIDFormat

**Pull request merged**